### PR TITLE
fix: add QueryClientProvider & pass pmv and app

### PR DESCRIPTION
**Pull request by ivy-lmu**
image

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -18,40 +18,40 @@
     "@jridgewell/gen-mapping" "^0.3.0"
     "@jridgewell/trace-mapping" "^0.3.9"
 
-"@axonivy/config-editor@~11.2.0-next.28":
```
Comment (weissreto) [inline yarn.lock:21]: Do we need to have this file to be in the repository? Seems to be house keeping for yarn.

File: vscode-extensions/inscription/webview/src/index.tsx [hunk 2]
```diff
@@ -78,13 +79,21 @@ function startInscriptionClient(webSocketAddress: string) {
 }
 
 export function render(inscriptionClient: InscriptionClient, selectedElement: SelectedElement = undefined): React.ReactElement {
+  const queryClient = new QueryClient();
```
Comment (weissreto) [inline vscode-extensions/inscription/webview/src/index.tsx:82]: It seems to be complex to integrate the inscription view app now. We need to know stuff about the react-query. 
I would like to see such things hidden in the inscription view app itself. 
Also the configuration with the three different parameters could be simplified by providing the selectedElement in one arg directly.

**Review by weissreto — APPROVED** [review]
Some architectural comments that can be solved in later PRs.
Good to merge to fix the broken integration now.

File: yarn.lock (hunk 1 above)
Comment (ivy-lmu) [reply yarn.lock:21]: yes, we want to have that file in the repo to get consistent installs across machines. see https://classic.yarnpkg.com/lang/en/docs/yarn-lock/

yarn.lock has changed slightly as I increased our own dependencies like @axonivy/config-editor and added new ones. if no dependencies are touched, it does not change.

File: vscode-extensions/inscription/webview/src/index.tsx (hunk 2 above)
Comment (ivy-lmu) [reply vscode-extensions/inscription/webview/src/index.tsx:82]: good ideas

Comment (ivy-lli) [reply vscode-extensions/inscription/webview/src/index.tsx:82]: I agree, that this it is not ideal that the vscode extension needs to know this stuff here! 
We definitly should move this more into the inscription view code, so this stuff here is not needed.
I wasn't aware of this, sorry.

**Pull request merged**